cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java

```diff
@@ -110,7 +109,7 @@
 
     private final String description;
 
-    private Action(String description) {
```

Comment:
enum constructors are always private. The JLS recommends not to declare them as private because that is redundant. Our checkstyle also warns about that.